README.md

```diff
@@ -1196,8 +1196,8 @@ public class DependsOnSample {
 
 ## Samples
 
-Examples of use AEM Authoring Toolkit API and DependsOn library are presented in a separate [AAT Samples](./aem-authoring-toolkit-samples) module.
+Examples of use AEM Authoring Toolkit API and DependsOn library are presented in a separate [AAT Samples](samples) module.
```

Comment:
`Examples of how to use,,,`
And also please name the link [samples], same wasy as the link at line 13 above

Reply:
done